company_from_url/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 AppsToGROW - Henrik Norlin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, exceptions, fields, models
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    access_url = fields.Char(string="Direct access URL")
```

Comment:
what you fill inhere is not actually an URL but a domain name. If you can, the addon itself should be renamed too

Reply:
`company_from_domain_name` or `web_company_from_domain_name` should be better then.

Reply:
Have renamed this.

Reply:
Or how about `multicompany_from_domain_name` ?

Reply:
whatever you prefer as long as the name doesn't lie. Some repo maintainers indeed prefer the name of the repo as prefix (that would count against `web`), others including me choose the name based on the addon the current one is an extension/modifiction of, that would then make it `base_company_from_domain_name`. If in doubt, ask Pedro ;-)

Reply:
@pedrobaeza What do you think? `multicompany_from_domain_name` sounds most descriptive to me